### PR TITLE
BB-417 fix(revision-table) : remove unneccesary div tag and use 'pull-right'

### PR DESCRIPTION
**Pull request by prabalsingh24**
### Problem
https://tickets.metabrainz.org/projects/BB/issues/BB-417
For the revisions table, right margin was lesser than left. It was looking unsymmetrical 
In the main page, `see all revisions` button was showing on the left side for smaller screens( mobile)

### Solution
Removed an unnecessary `<div id=container>` (made it same as search result table)
For the main page, used `pull-right` for `see all revisions` button 

Screenshot from 2020-02-09 18-07-31

Screenshot from 2020-02-09 18-07-39

### Areas of Impact
 src/client/components/pages/index.js
 src/client/components/pages/revisions.js

### CODE REVIEW

File: src/client/components/pages/index.js [hunk 1]
```diff
@@ -212,20 +212,17 @@ class IndexPage extends React.Component {
...
-								See all revisions
-							</Button>
-						</Col>
-					</Row>
+					<RevisionsTable
+						results={this.props.recent}
+					/>
+					<Button
+						bsStyle="primary"
+						className="pull-right"
```
Comment (MonkeyDo) [inline src/client/components/pages/index.js]: I think the button would be better if it was centered.
On mobile it works OK on the right, but on desktop size I feel it's not visible enough and I find myself looking for it.

You can wrap a `<div className="text-center">` around the button to center it.
What do you think?

Comment (prabalsingh24) [reply src/client/components/pages/index.js]: Like this?

Screenshot from 2020-02-12 08-02-24

Comment (MonkeyDo) [reply src/client/components/pages/index.js]: I think that works better yes. My eye naturally finds it at the end of the table

Comment (prabalsingh24) [reply src/client/components/pages/index.js]: > I think that works better yes. My eye naturally finds it at the end of the table

I'll make the change

**Pull request merged**